https://github.com/antirez/redis/pull/4390

From 8832af49fa1f4fdd8f9e152f013bb71b343d26dd Mon Sep 17 00:00:00 2001
From: "Bernhard M. Wiedemann" <bwiedemann@suse.de>
Date: Sun, 22 Oct 2017 05:20:38 +0200
Subject: [PATCH 1/2] Use RELEASENOTES date instead of build date

in order to make builds reproducible.
See https://reproducible-builds.org/ for why this is good.
---
 src/mkreleasehdr.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: redis-5.0.5/src/mkreleasehdr.sh
===================================================================
--- redis-5.0.5.orig/src/mkreleasehdr.sh
+++ redis-5.0.5/src/mkreleasehdr.sh
@@ -1,7 +1,7 @@
 #!/bin/sh
 GIT_SHA1=`(git show-ref --head --hash=8 2> /dev/null || echo 00000000) | head -n1`
 GIT_DIRTY=`git diff --no-ext-diff 2> /dev/null | wc -l`
-BUILD_ID=`uname -n`"-"`date +%s`
+BUILD_ID=${HOST:-`uname -n`}"-"`date -r ../00-RELEASENOTES +%s`
 if [ -n "$SOURCE_DATE_EPOCH" ]; then
   BUILD_ID=$(date -u -d "@$SOURCE_DATE_EPOCH" +%s 2>/dev/null || date -u -r "$SOURCE_DATE_EPOCH" +%s 2>/dev/null || date -u %s)
 fi
